File: app/src/main/java/org/techtown/deulline_android/retrofit/dto/productInfoVO.java
package org.techtown.deulline_android.retrofit.dto;

import com.google.gson.annotations.SerializedName;

public class productInfoVO {
    @SerializedName("productId")
    private Long productId;

    @SerializedName("productName")
    private String productName;

    @SerializedName("price")
    private int price;

    @SerializedName("reviewCount")
    private int reviewCount;

    @SerializedName("ranking")
    private int ranking;

    public Long getProductId() {
        return productId;
    }

    public String getProductName() {
        return productName;
    }

    public int getPrice() {
        return price;
    }

    public int getReviewCount() {
        return reviewCount;
    }

    public int getRanking() {
        return ranking;
    }
}

File: app/src/main/java/org/techtown/deulline_android/network/dto/ProductInfoVO.kt
package org.techtown.deulline_android.network.dto

import com.google.gson.annotations.SerializedName

data class ProductInfoVO (
    @SerializedName("productId")
    val productId : Long,

    @SerializedName("productName")
    val productName : String,

    @SerializedName("price")
    val price : Int,

    @SerializedName("reviewCount")
    val reviewCount : Int,

    @SerializedName("ranking")
    val ranking : Int
)